test/observable_list_test.dart

```diff
@@ -269,6 +269,16 @@ _runTests() {
       });
     });
 
+    test('sort or 2 elements from reaction fn', () {
+      ObservableList list = toObservable([3, 1]);
+      var futureSub = list.listChanges.listen(_reSort(list));
+      list.sort();
+      expect(list, [1, 3]);
+      // Delay clean up, because terminating listeners immediately hides bug.
```

Comment:
hmmmmm. That's odd. It might be worth debugging into it and seeing what's going on. That suggests that it didn't record any changes, which seems wrong (sorting the list should yield [1, 3] right?)